pkgs/tools/misc/less/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "less";
-  version = "600";
+  version = "608";
 
   src = fetchurl {
     url = "https://www.greenwoodsoftware.com/less/less-${version}.tar.gz";
-    sha256 = "sha256-ZjPWqis8xxevssIFd4x8QsRiD2Ox1oLz0SyYrwvnTSA=";
+    sha256 = "1i75fgbmjl0i1q67f4322xmn1ablim4j15xv95cq27bcvs7fni4n";
```

Hash is wrong.

https://logs.nix.ci/?key=nixos/nixpkgs.189974&attempt_id=bad859ad-ef80-4749-babe-a54ad0ae0a90

```suggestion
    sha256 = "02f2d9d6hyf03va28ip620gjc6rf4aikmdyk47h7frqj18pbx6m6";
```

---

...weird. nix-prefetch-url gives me:

```
$ nix-prefetch-url http://greenwoodsoftware.com/less/less-608.tar.gz
path is '/nix/store/d3dfkz440409ads5x7hgin1lvwyrkjpj-less-608.tar.gz'
1i75fgbmjl0i1q67f4322xmn1ablim4j15xv95cq27bcvs7fni4n
$ nix-prefetch-url --unpack http://greenwoodsoftware.com/less/less-608.tar.gz
path is '/nix/store/qgv4r91kpzv1i0bsbapzzm0045kb9xyb-less-608.tar.gz'
0jf2cqbrjryq005hc0c6cy25anh5ywmmdxl821y4hdrzbhjca4cc
```

---

oh, ffs. I think nix-prefetch-url removes the gzip layer.

it's served with `Content-Type: application/x-gzip`, though, so really that's not incorrect behaviour...

---

> nix-prefetch-url gives me ...

That works for `fetchzip`, this package uses `fetchurl`.